dotCMS/src/main/java/com/dotmarketing/startup/StartupAPI.java

```diff
@@ -0,0 +1,47 @@
+package com.dotmarketing.startup;
+
+import com.dotmarketing.exception.DotDataException;
+import com.dotmarketing.exception.DotRuntimeException;
+
+import java.util.List;
+
+/**
+ * API to interact with the start up tasks.
+ *
+ * @author jsanca
+ */
+public interface StartupAPI {
```

Comment:
I would name it `StartupTaskAPI` is very specific to startup tasks and just `Startup` could create confusion about the purpose of the class.